code/modules/jobs/job_types/ncr.dm

```diff
@@ -347,6 +347,73 @@ Weapons		Service Rifle, Rockwell, 9mm pistol, all good.
 		/obj/item/melee/onehanded/knife/trench = 1,
 		)
 
+/datum/job/ncr/f13representative
+	title = "NCR Representative"
+	flag = F13REP
+	total_positions = 1
+	spawn_positions = 1
+	description = "You are a representative for the NCR. You are here to further the objective of the NCR, have been supplied with ample amounts of money to do so."
+	supervisors = "The Captain and the NCR"
+	display_order = JOB_DISPLAY_ORDER_REPRESENTATIVE
+	outfit = /datum/outfit/job/ncr/f13representative
+
+	loadout_options = list(
+		/datum/outfit/loadout/repbrahminbaron,
+		/datum/outfit/loadout/repambassador,
+		/datum/outfit/loadout/repgunrunner
+		)
+
+	matchmaking_allowed = list(
+		/datum/matchmaking_pref/friend = list(
+			/datum/job/ncr
+			),
+		/datum/matchmaking_pref/rival = list(
+			/datum/job/ncr
+			)
+		)
+
+/datum/outfit/job/ncr/f13representative
+	name = "NCR Representative"
+	uniform = /obj/item/clothing/under/f13/ncr/ncr_dress
+	jobtype	= /datum/job/ncr/f13representative
+	id = /obj/item/card/id/dogtag/ncrtrooper
```

Comment:
I suggest replacing the NCR dogtag with an Ambassador ID, to show a little difference